codejam3-pokemon/Program.cs

```diff
@@ -148,10 +147,42 @@ public static async Task GetPokemon(string pokemonspecies)
        
     }
 
-    public static void PrintPoke(string pokemonspecies){
+    public static async Task<bool>  isRealPoke(string pokemonspecies)
+    {
+        try
```

Comment:
this method to check if the pokemon species was found is good since you do not move to other operations if the pokemon species entered is invalid.